packages/legend-graph/src/graph/BasicModel.ts

```diff
@@ -614,10 +614,7 @@ export abstract class BasicModel {
    * expose it. The other benefit of exposing this in the main graph is that we could
    * do better duplicated path check
    */
-  protected renameOwnElement(
-    element: PackageableElement,
-    newPath: string,
-  ): void {
+  renameOwnElement(element: PackageableElement, newPath: string): void {
```

Comment:
why are we removing the `protected` part of this. The NOTE explains that the main graph `PureModel`, which is the only mutable graph, is the only one that can expose this and should use this.